Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/zh_CN/Command-Line-Options.po:
```diff
@@ -1,75 +1,75 @@
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "维基"
```
```suggestion
msgstr "wiki"
```

ignotus666 replied:
Corrected on weblate

yangyangdaji commented:
i'm ready

yangyangdaji commented:
OK

ignotus666 commented on _translator-files/po/zh_CN/FAQ.po:
```diff
@@ -46,159 +47,161 @@ msgid ""
 #: ../wiki/en/FAQ.md:28
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "还请记住，所有参与者都应遵循 [黄金法则](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) 这也将决定您是否可以使用及时正确。"
```
This is an example of one of those links. The part after the hashtag should be translated, using the exact same wording as the heading in `Client-Troubleshooting`. On the other hand, I don't have a clue about how punctuation works in Chinese - there don't seem to be spaces - so I'm not sure if everything should be joined up by hyphens or they're only inserted where there are commas or other punctuation marks. Once I get the website working I'll look into it.

ignotus666 replied:
Hmm, I'm struggling with this; please don't change these links yet until I figure out how they have to be done in Chinese.

ignotus666 replied:
Ok, so the above link works when written as follows:

`[黄金法则](/wiki/Client-Troubleshooting#你们听起来都不错但很难保持在一起)`. No hyphens.

ignotus666 commented on _translator-files/po/zh_CN/1-index.po:
```diff
@@ -35,24 +35,39 @@ msgid ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
+"lang: \"zh_CN\"\n"
+"永久性的。/\n"
```
This should be left as `permalink:`

yangyangdaji replied:
Fixed. Wait for sync

ignotus666 commented on _translator-files/po/zh_CN/Installation-for-Windows.po:
```diff
@@ -1,289 +1,290 @@
 #: ../wiki/en/Installation-for-Windows.md:28
 msgid "If you don't want to use ASIO or prefer [JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html), you can also use the [JACK version of Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})."
-msgstr ""
+msgstr "如果您不想使用 ASIO 或者更喜欢 [JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html)，也可以使用 [JACK 版本的 Jamulus]({{ site.download_root_link } {{ site.download_file_names.windows-jack }}）。"
```
There's a closing curly bracket missing in `{{ site.download_root_link }`.

yangyangdaji replied:
Fixed For now

ignotus666 commented on _translator-files/po/zh_CN/Running-a-Server.po:
```diff
@@ -173,16 +176,20 @@ msgid ""
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Registered Server\">\n"
+"<figcaption>How Registered Servers work</figcaption>\n"
```
Preferably, the figcaption text should be translated.

yangyangdaji replied:
done

ignotus666 commented on _translator-files/po/zh_CN/Running-a-Server.po:
```diff
@@ -193,111 +200,115 @@ msgid ""
 #: ../wiki/en/Running-a-Server.md:88
 msgid "Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or the [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }})"
-msgstr ""
+msgstr "下载 [最新的无头 (amd64) .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) 或者，如果您使用 Raspberry Pi 等，请下载 [最新的 armhf .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) 或 [最新的 arm64 .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless- arm64 }})"
```
There's an unwanted space in `site.download_file_names.deb-headless- arm64` just before `arm64`.

yangyangdaji replied:
ok

ignotus666 commented on _translator-files/po/zh_CN/1-index.po:
```diff
@@ -35,24 +35,39 @@ msgid ""
+"permalink: /\n"
+"layout: mainhomepage\n"
+"ldjAppCategory: \"音乐\"\n"
+"ldjSlogan:\"在线播放音乐。与朋友一起。免费的。\"\n"
```
There should be a space after the colon.

yangyangdaji replied:
OK

yangyangdaji replied:
Wait for sync

ignotus666 commented on _translator-files/po/zh_CN/1-index.po:
```diff
@@ -35,24 +35,39 @@ msgid ""
 "---\n"
 msgstr ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
```
Jekyll keeps complaining about this line but I can't see what's wrong with it. The error message is:
`did not find expected key while parsing a block mapping at line 2 column 1`
@ann0see any ideas?

ignotus666 replied:
Ok, got it. Missing quotation marks further below.

ignotus666 commented on _translator-files/po/zh_CN/1-index.po:
```diff
@@ -35,24 +35,39 @@ msgid ""
+"mAltProgIcon: \"Jamulus 图标\"\n"
+"mTSlogan: \"在线播放音乐。与朋友一起。免费的。\"\n"
+"mTGetStartedNow: \"现在就开始吧！\"\n"
+"mTDownloadNow: \"现在下载\n"
```
Missing quotation mark at the end of `现在下载`. The original in EN uses single quotes here - not sure if it's important but no harm in keeping it that way.

yangyangdaji replied:
ok

ignotus666 commented on _translator-files/po/zh_CN/1-index.po:
```diff
@@ -35,24 +35,39 @@ msgid ""
+"mTSlogan: \"在线播放音乐。与朋友一起。免费的。\"\n"
+"mTGetStartedNow: \"现在就开始吧！\"\n"
+"mTDownloadNow: \"现在下载\n"
+"mTOtherPlatforms: \"其他平台\n"
```
Closing quote missing here too.

This is the last error I get and when corrected the website works fine in Chinese : ) All that's left are the links, which I'll test now to see how they should be done. Oh, and screenshots, which would be cool to have too. Just ask if in doubt about how to contribute them.

yangyangdaji replied:
OK

ignotus666 commented on _translator-files/po/zh_CN/Software-Manual.po:
```diff
@@ -91,364 +94,368 @@ msgid ""
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"<img src=\"{% include img/en-screenshots/main-screen-medium.inc %}\" style=\"border: 5px solid gray;\" loading=\"lazy\" alt=\"Jamulus 主窗口的图像\">\n"
```
Sorry, I overlooked this - you need to change the language code in `en-screenshot` to `zh_CN-screenshot`. That's for all screenshots.

yangyangdaji replied:
I also know, but may not find the file.

ignotus666 replied:
Yes, it will. The zh_CN folder for screenshots contains the ones in EN until they are replaced.

yangyangdaji replied:
done

ignotus666 commented on _translator-files/po/zh_CN/Include-Server-Commands.po:
```diff
@@ -1,111 +1,111 @@
 #: ../wiki/en/Include-Server-Commands.md:17
 msgid "`-u` or `--numchannels` Maximum number of channels (Clients)"
-msgstr ""
+msgstr "“-u”或“--数字通道” 最大通道数（客户端）"
```
Wrong quotes used.

yangyangdaji replied:
Fixed

ignotus666 commented on _translator-files/po/zh_CN/Include-Shared-Commands.po:
```diff
@@ -1,81 +1,81 @@
 #, no-wrap
 msgid "`-h` or `--help`           Display help text         \n"
-msgstr ""
+msgstr "“-h”或“--help” 显示帮助文本         \n"
```
Quotes

yangyangdaji replied:
Fixed

ignotus666 commented on _translator-files/po/zh_CN/Software-Manual.po:
```diff
@@ -91,364 +94,368 @@ msgid ""
 #: ../wiki/en/Software-Manual.md:94
 msgid "<figure><img src=\"{% include img/en-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"Image of a pair of server mixer controls\"></figure>"
-msgstr ""
+msgstr "<figure><img src=\"{% include img/zh_CN-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"一对服务器混音器控件的图像/figure>"
```
`\"><` missing at end of line before `/figure`.

ignotus666 commented on _translator-files/po/zh_CN/Software-Manual.po:
```diff
@@ -91,364 +94,368 @@ msgid ""
 #: ../wiki/en/Software-Manual.md:94
 msgid "<figure><img src=\"{% include img/en-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"Image of a pair of server mixer controls\"></figure>"
-msgstr ""
+msgstr "<figure><img src=\"{% include img/zh_CN-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"一对服务器混音器控件的图像></figure>"
```
There's still a closing quotation mark missing.

ignotus666 commented on _translator-files/po/zh_CN/Client-Troubleshooting.po:
```diff
@@ -46,236 +47,238 @@ msgid ""
 #, no-wrap
 msgid "**Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4ALL setup section](Installation-for-Windows#setting-up-asio4all)\n"
-msgstr ""
+msgstr "**Windows 用户 (ASIO4ALL)**:如果您使用的是 ASIO4ALL 驱动程序，请查看 [ASIO4ALL 设置部分](Installation-for-Windows#设置 ASIO4ALL)\n"
```
In these links there can't be any capital letters nor spaces. It seems that if all the characters are Chinese, as there are no spaces, hyphens aren't required either. But in this case, as there's a space, it should be replaced with a hyphen - so it should look like this:

`(Installation-for-Windows#设置-asio4all)`.

These links are where most people stumble, and add Chinese characters into the mix and it gets even more complicated ;)

ignotus666 commented on _translator-files/po/zh_CN/Client-Troubleshooting.po:
```diff
@@ -46,236 +47,238 @@ msgid ""
 #: ../wiki/en/Client-Troubleshooting.md:80
 msgid "In some cases, it may be your ISP that is blocking your use of Jamulus. See the note on the [server troubleshooting page](Server-Troubleshooting#nobody-can-connect-to-my-server---but-i-can-connect-locally) about \"Nobody can connect to my Server\""
-msgstr ""
+msgstr "在某些情况下，可能是您的 ISP 阻止了您使用 Jamulus。请参阅 [服务器故障排除页面](Server-Troubleshooting#没有人可以连接到我的服务器 - 但我可以在本地连接)上关于“没有人可以连接到我的服务器”的注释"
```
No spaces allowed in link. It should look like this:

`(Server-Troubleshooting#没有人可以连接到我的服务器---但我可以在本地连接)`

ignotus666 replied:
Corrected this myself.

ignotus666 commented on _translator-files/po/zh_CN/Server-Troubleshooting.po:
```diff
@@ -51,113 +52,115 @@ msgid ""
 #: ../wiki/en/Server-Troubleshooting.md:41
 msgid "This should be your **public** IP address (find that with [Google](https://www.google.com/search?q=whatsmyip)). Connect your **own** Jamulus Client to the **local** IP address of your Server (`localhost` or `127.0.0.1` if it's on the same machine as your Client). Note that your public IP address can change - see the note on \"dynamic DNS\" in [this guide](Running-a-Server#dynamic-dns-and-w ...
-msgstr ""
+msgstr "这应该是您的**公共** IP 地址（通过 [Google](https://www.google.com/search?q=whatsmyip) 查找）。将您的**自己的** Jamulus 客户端连接到服务器的**本地** IP 地址（`localhost` 或`127.0.0.1`，如果它与您的客户端在同一台机器上）。请注意，您的公共 IP 地址可能会更改 - 请参阅 [本指南] 中有关“动态 DNS”的说明（运行服务器#动态 DNS 以及您可能需要它的原因）。"
```
The structure for links is: `[some text](A-Document#paragraph-header)`. The content in square brackets should not be separated from the link in normal brackets. I don't know how you would have to reformulate the sentence to maintain this rule. Also, spaces in`（运行服务器#动态 DNS 以及您可能需要它的原因）` should be replaced with hyphens.

ignotus666 replied:
What I mean is that it should look like this:

`请注意，您的公共 IP 地址可能会更改 - 请参阅 [本指南](Running-a-Server#运行服务器#动态-DNS-以及您可能需要它的原因) 中有关“动态 DNS”的说明。` 

It's missing `Running-a-Server#` (isn't it?)

ignotus666 replied:
Corrected this too.

ann0see commented:
@ignotus666 @yangyangdaji This still seems to be blocking. When can we plan the release?

ignotus666 commented:
I think it's just these last couple of corrections that are missing to make it 100% (or near enough), though I'm not sure if there are many more of those links to update. The website works fine as it is though.

ann0see commented:
Ok. Thanks for testing. It probably makes sense to wait on the link fixes then. But feel free to merge this PR.

yangyangdaji commented:
@ignotus666 @ann0see I found some problems already on fixes for now on Weblate 
Wait for sync and review

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Installation-for-Linux
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-linux/

ignotus666 commented:
Ok, so I did another quick review and corrected a few things I found myself to speed things up. I'll merge it now so the website can go live when you see fit @ann0see.

@yangyangdaji when it goes live (should be soon) it would be great if you could go through the website to check if everything works (links) and is displayed correctly (formatting). I think it all looks ok but since I can't read Chinese I can't really be sure.

yangyangdaji commented:
ok @ignotus666 Thanks for helping me

yangyangdaji commented:
@ignotus666 how do test

ignotus666 commented:
You'll have to either wait until the website goes live, or you can test locally - but you will need to install Jekyll. You can find instructions [here](https://github.com/jamulussoftware/jamuluswebsite#viewing-the-website-offline).

yangyangdaji commented:
ok
